lib/trento/charts/hosts/host_memory_chart.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Trento.Charts.Hosts.HostMemoryChart do
+  @moduledoc """
+  Represents Memory chart data for a single host
+
+  The chart has different series
+    - ram_total
+    - ram_used
+    - ram_cache_and_buffer
+    - ram_free
+    - swap_used
+  """
+
+  alias Trento.Charts.ChartTimeSeries
+
+  @enforce_keys [:ram_total, :ram_used, :ram_cache_and_buffer, :ram_free, :swap_used]
+  defstruct [:ram_total, :ram_used, :ram_cache_and_buffer, :ram_free, :swap_used]
```

Comment:
we could sort here also by alphabet, but up 2 you :) 
` defstruct [ :ram_cache_and_buffer,  :ram_free, :ram_total,  :ram_used,  :swap_used
  ]`